test/controllers/projects_controller_test.rb

```diff
@@ -421,10 +448,19 @@ def test_destroy_project_other
     destroy_project_helper(projects(:bolete_project), rolf)
   end
 
+  # prove that mere member cannot destroy project
   def test_destroy_project_member
-    eol_project = projects(:eol_project)
-    assert(eol_project.is_member?(katrina))
-    destroy_project_helper(eol_project, katrina)
+    project = projects(:rolf_project)
+    member = users(:katrina)
+    assert(
+      project.member?(member) && project.user != member &&
+      !project.admin?(member) &&
+      NameDescription.where(source_name: project.title).any?,
+      "Bad fixtures: member must be project member, but not owner or admin " \
+      "and project must be the source of a NameDescription"
+    )
+
+    destroy_project_helper(project, katrina)
```

Comment:
`destroy_project_helper` 
- Needs a Project with an associated NameDescription
- Should not destroy the Project, so it must be called with a user who cannot destroy.